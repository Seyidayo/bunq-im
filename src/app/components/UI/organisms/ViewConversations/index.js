export { default } from './Conversations';
export * from './Conversations';
