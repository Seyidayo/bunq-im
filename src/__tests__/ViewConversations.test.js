import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

import ViewConversations from 'app/components/UI/organisms/ViewConversations';

describe('<ViewConversations />', () => {
  it('It renders correctly and matches snapshot', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <ViewConversations />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
