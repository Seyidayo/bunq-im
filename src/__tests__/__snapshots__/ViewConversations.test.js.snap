// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewConversations /> It renders correctly and matches snapshot 1`] = `
<section
  className="conversation__wrapper"
>
  <aside>
    <header
      className="header"
    >
      <h2
        className="text-orange h4"
      >
        Conversations
      </h2>
    </header>
  </aside>
</section>
`;
